src/ptvsd/_vendored/pydevd/pydevd.py

```diff
@@ -806,10 +807,12 @@ def apply_files_filter(self, frame, filename, force_check_project_scope):
         try:
             return self._apply_filter_cache[cache_key]
         except KeyError:
+            debug_trace_level = DebugInfoHolder.DEBUG_TRACE_LEVEL
             if self.plugin is not None and (self.has_plugin_line_breaks or self.has_plugin_exception_breaks):
                 # If it's explicitly needed by some plugin, we can't skip it.
                 if not self.plugin.can_skip(self, frame):
-                    # print('include (include by plugins): %s' % filename)
+                    if debug_trace_level >= 3:
```

Comment:
it looks like that this pattern(if level ...) is repeated, if then, couldn't we put this as a method?

Reply:
Done